scripts/e2e.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 DigitalOcean
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+docker build -t do-ccm-e2e e2e
+docker run --rm -it -v ~/.ssh:/root/.ssh -v $PWD:/app -w /app --env-file e2e/.env do-ccm-e2e ./e2e/e2e.sh
```

Comment:
When I run the container as-is, it downloads all dependencies via Go modules. I suppose that's going to happen on every run since we don't mount in the cache?

I'd also like to be able to just use the vendor folder. Adding `-e GO111MODULE` to docker run disables Go modules in the container (as I have that variable set to `off`) but I get a build error afterwards. I suppose it's because the files aren't copied below the `GOPATH` or `/go`?

Reply:
I can get this to work by keeping go modules enabled and using `-mod=vendor` :+1: